ups-package-tracker/app/entrypoint.sh

```diff
@@ -0,0 +1 @@
+node src/package-tracker.js $@
```

Comment:
this needs a shebang (#!/bin/sh) on the first line. that what cause the error you get I presume.